utils/build/docker/python/flask/integrations/messaging/aws/sqs.py

```diff
@@ -0,0 +1,54 @@
+import logging
+import time
+
+import boto3
+
+
+def sqs_produce(queue, message):
```

Comment:
nit: you can create the name based on the test name using the pytest `requests` fixture, instead of passing it in every time

(example: https://github.com/DataDog/dd-trace-py/blob/main/tests/contrib/kafka/test_kafka.py#L63)

Reply:
this requires a queue argument since its also used by the weblog buddy cross tracer tests